components/automate-cli/pkg/verifyserver/services/softwareversionservice/softwareversionservice.go

```diff
@@ -0,0 +1,203 @@
+package softwareversionservice
+
+import (
+	"regexp"
+	"strconv"
+	"strings"
+
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/models"
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/utils/fiberutils"
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/utils/getosutils"
+	"github.com/chef/automate/lib/logger"
+	"github.com/pkg/errors"
+)
+
+type ISoftwareVersionService interface {
+	GetSoftwareVersionServices(string) (*models.SoftwareVersionDetails, error)
+}
+
+type SoftwareVersionService struct {
+	cmdCheckArray []string
+	osFilepath    string
+	logger        logger.Logger
+	CheckPath     func(cmd string) (string, error)
+}
+
+func NewSoftwareVersionService(logger logger.Logger, CheckPath func(cmd string) (string, error)) ISoftwareVersionService {
+	return &SoftwareVersionService{
+		cmdCheckArray: cmdCheckArray,
+		osFilepath:    OSFILEPATH,
+		logger:        logger,
+	}
+}
+
+const (
```

Comment:
Move these constants to constants folder. Where you can have a file for you check constants

Reply:
Done

Reply:
if you could have moved the constants, why these are still here.